addons/web/tests/test_db_manager.py

```diff
@@ -119,3 +137,26 @@ def test_database_duplicate(self):
         self.assertEqual(res.status_code, 303)
         self.assertIn('/web/database/manager', res.headers['Location'])
         self.assertDbs([])
+
+        # check if user is automatically logout after trying to access a deleted database.
+        # NOTE: the current behaviour is that the next request will crash, returning
+        # a response 500, but will also logout the user.
+        # If this changes and all sessions are cleaned with the database, we could remove
+        # the request on /web before checking the session content
```

Comment:
Should be part of the commit message rather than a comment.

Reply:
Or both a commit message and a comment ;)